tournaments/index.js

```diff
@@ -950,13 +953,13 @@ function createTournament(room, format, generator, playerCap, isRated, args, out
 	}
 	format = Dex.getFormat(format);
 	if (format.effectType !== 'Format' || !format.tournamentShow) {
-		output.errorReply(format.id + " is not a valid tournament format.");
-		output.errorReply("Valid formats: " + Object.values(Dex.formats).filter(f => f.tournamentShow).map(format => format.name).join(", "));
+		output.errorReply(`${format.id} is not a valid tournament format.`);
+		output.errorReply(`Valid formats: ${Object.values(Dex.formats).filter(f => f.tournamentShow).map(format => format.name).join(', ')}`);
```

Comment:
`Object.values(Dex.formats).filter(f => f.tournamentShow).map(format => format.name).join(', ')`

should probably be a temporary variable.